.github/workflows/slsa-goreleaser.yml

```diff
@@ -0,0 +1,55 @@
+name: Test SLSA
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - "*" 
+
+permissions: read-all
```

Comment:
Note: this can live in the same goreleaser.yml, but I separate it to help people see what a standard SLSA workflow looks like.